Sources/swift-container-plugin/Documentation.docc/run.md

```diff
@@ -0,0 +1,20 @@
+# Run your service
+
```

Comment:
```suggestion

Run the container you built on cloud infrastructure or locally on a desktop container runtime.
```

^ In an article - the first sentence under the `#` header is the "abstract" - it's a single, typically imperative, sentence that is exposed in DocC in curation pages to give readers a hint (beyond the title) of what's in the article.